integration-tests/src/test/scala/play/api/libs/ws/akkahttp/AkkaHttpWSClientSpec.scala

```diff
@@ -0,0 +1,16 @@
+package play.api.libs.ws.akkahttp
+
+import org.specs2.concurrent.ExecutionEnv
+import org.specs2.execute.Result
+import play.api.libs.ws.{ StandaloneWSClient, WSClientSpec }
+
+class AkkaHttpWSClientSpec(implicit override val executionEnv: ExecutionEnv) extends WSClientSpec {
+  def withClient()(block: StandaloneWSClient => Result): Result = {
+    val client = StandaloneAkkaHttpWSClient()
+    try {
+      block(client)
+    } finally {
+      client.close()
+    }
+  }
+}
```

Comment:
Yea. Tests are shared between the backends, but are different between the Scala and Java APIs.

It might be possible to share the tests between the Scala and Java APIs as well, but it would need another abstraction layer. Maybe implemented using FreeMonad. :)